pkg/services/smtp/smtp.go

```diff
@@ -59,15 +60,26 @@ func (service *Service) Send(message string, params *types.Params) error {
 	if params == nil {
 		params = &types.Params{}
 	}
-	client, err := getClientConnection(service.config.Host, service.config.Port)
+	client, err := getClientConnection(service.config.Host, service.config.Port, service.config.Encyption)
```

Comment:
```suggestion
	client, err := getClientConnection(service.config)
```

This would require some updates of the function as well, but I think you get the gist. 😅 In the spirit of maintainability, we should refrain from implementing triadic functions if possible.

Reply:
It's for testing purposes I think, but I'll look into it.

Reply:
Nope. Changed to just requiring `config`.